presto-spi/src/main/java/io/prestosql/spi/statistics/ColumnStatistics.java

```diff
@@ -114,6 +114,12 @@ public static Builder builder()
         return new Builder();
     }
 
+    /**
+     * If one of the estimates below is unspecified, the default "unknown" estimate value
+     * (represented by floating point NaN) may cause the resulting symbol statistics
+     * to be "unknown" as well.
+     * @see SymbolStatsEstimate
```

Comment:
`SymbolStatsEstimate` is not part of SPI, but rather part of engine, so connectors shouldn't assume any specific behavior.

The method that determines if `SymbolStatsEstimate` is unknown is `io.prestosql.cost.SymbolStatsEstimate#isUnknown`.

Reply:
I agree with your comment, but in this PR, I was trying to explain the engine's expectations from the connector statistics' estimation API, following https://github.com/prestodb/presto/pull/12177#issue-242261898.
Is it possible to change this comment to explain it better (e.g. what happens when some of the statistics are not available)?

Reply:
OTOH, we can add a link to the documentation (being added by https://github.com/prestosql/presto/pull/127) and explain it there. What do you think?

Reply:
Hm.. Maybe we could have a section there that describes how connectors should behave? WDYT: @martint @findepi ?

Reply:
It’s important for that kind of information to be in the API and accessible from the IDE. People don’t read manuals. Also, any API documentation that is not part of the code is even more likely to get stale as we continue to change the code.

Let’s keep it here but remove the reference to the non-spi class and phrase it in terms of how the engine may not be able to derive stats if any of the provided stats is unknown.

Reply:
Sounds good - pushed an updated commit 23cc24b523ba7d5b8f1284042fee980bc09fede7.